eos-paygd/main.c

```diff
@@ -487,6 +569,23 @@ main (int   argc,
       eospayg_efi_root_pivot ();
     }
 
+  /* Use /bin/mkdir instead of mkdir() to ensure the mode is unaffected by
+   * the process's umask.
+   */
+  system_ret = system ("/bin/mkdir -p --mode=755 " LOGFILE_DIRNAME);
```

Comment:
Is this not done by the systemd-tmpfiles snippet? Sorry if that’s already been covered in a discussion; I only skimmed through them.

*Edit*: Oh, I see it was [discussed here](https://github.com/endlessm/eos-payg/pull/104#discussion_r854633924). Perhaps the details about startup ordering and dependencies on `RequiresMountsFor=` are worth a code comment here?